src/tapis-app/src/Sections/Apps/Apps.tsx

```diff
@@ -13,20 +19,20 @@ const Apps: React.FC = () => {
     [ setSelectedApp ]
   )
 
-  const list = <AppsListing />
-
-  const detail = (
-    <div>
-      {selectedApp ? selectedApp.id : "Select an app from the list"}
-    </div>
-  )
-
   return (
-    <ListSection
-      name="Apps"
-      list={list}
-      detail={detail}
-    />
+    <ListSection>
+      <ListSectionHeader>Apps</ListSectionHeader>
+      <ListSectionBody>
```

Comment:
If we're going to include `ListSection` in some of the main components should `ListSection` be moved into `tapis-ui/src/_common/`, or should we import it into whatever tapis-ui component from here? Also wondering if the ListSection, Header, Body pieces could be imported into the main App to wrap around each component there. What do you think?

Reply:
I think they should be part of tapis-app since the layout is specific to our application. (And not the best layout, either...)